src/Humanizer/EnumHumanizeExtensions.cs

```diff
@@ -23,6 +24,26 @@ public static class EnumHumanizeExtensions
         public static string Humanize(this Enum input)
         {
             var type = input.GetType();
+
+            if (IsBitFieldEnum(input) && !DirectlyMapsToEnumConstant(input))
+            {
+                var inputIntValue = Convert.ToInt32(input);
+
+                var humanizedEnumValues = new List<string>();
+
+                foreach (var enumValue in Enum.GetValues(type))
+                {
+                    var enumIntValue = (int)enumValue;
+
+                    if ((inputIntValue & enumIntValue) != 0)
+                    {
+                        humanizedEnumValues.Add(EnumHumanizeExtensions.Humanize((Enum)enumValue));
+                    }
+                }
+
+                return String.Join(", ", humanizedEnumValues);
```

Comment:
Can we use `CollectionHumanizeExtensions` here?
The output of the test would currently change to `Red and Dark Gray` on English locale, but it would be localized.
Note: The test class have to inherit from `AmbientCulture` like `CollectionHumanizeTests` does.